tests/jvm/src/test/resources/metac.expect

```diff
@@ -1674,10 +1963,6 @@ Synthetics:
   Int => scala/Int#
 [19:15..19:22): message => string2Number(*)
   string2Number => example/ImplicitConversion#string2Number().
-[22:2..22:2):  => *.apply
```

Comment:
This change looks unexpected, can you see what is happening?

Reply:
Fixed.